backend/alloc/alloc.go

```diff
@@ -29,9 +29,10 @@ func New(sm subnet.Manager, network string) backend.Backend {
 	}
 }
 
-func (m *AllocBackend) Init(extIface *net.Interface, extIP net.IP) (*backend.SubnetDef, error) {
+func (m *AllocBackend) Init(extIface *net.Interface, extIaddr net.IP, extEaddr net.IP) (*backend.SubnetDef, error) {
 	attrs := subnet.LeaseAttrs{
-		PublicIP: ip.FromIP(extIP),
+		InterfaceIP: ip.FromIP(extIaddr),
+		EndpointIP: ip.FromIP(extEaddr),
```

Comment:
`InterfaceIP` is good but let's change `EndpointIP` to `PublicIP`. I would actually prefer `ExternalIP` but I'd rather stay with what's currently stored in etcd.